libs/shared/package.json

```diff
@@ -26,7 +26,7 @@
     "dist/"
   ],
   "dependencies": {
-    "axios": "1.2.0",
+    "axios": "^1.4.0",
```

Comment:
the Axios version 1.2.0 was buggy, updating it to the latest helped solve the original issue, but...